utils/errors.ts

```diff
@@ -0,0 +1,9 @@
+export function errorFilter(value: unknown | Error): boolean {
+  if (value instanceof Error) {
+    // TODO: Have a proper error reporting mechanism
+    console.log(`Errored at`);
+    console.log(value);
```

Comment:
do you still wnat to log these

Reply:
I'd say yes as until we don't have error reporting we can try to reproduce misconfigured synths and fix them in the CMS